Templates/sample_inventory/README.md

```diff
@@ -0,0 +1,155 @@
+the `Sample_Metadata` sheet on the
+`Sample_Inventory_Metadata_2021Q2_template.xlsx`:
+
+
+#### **Sample ID** (string)
+* The unique identifier of the biological sample. See the previous
+`Usage of sample identifiers` section for more detailed instructions for
+this field.
+
+#### **Sample Type** (string)
+* The specimen type of this sample. The list of permitted values are found in 
+the `Specimen Type` box in the `Controlled Values` tab.
+
+#### **Species** (string)
+* Common name for the species of the sample. The list of permitted values can
+be found in the `Controlled Values` sheet - currently allowed values
+are `human`, `mouse`, or `marmoset`.
+
+#### **Species NCBI Taxonomy ID** (string)
+* The unique identifier for the species of the sample. A link to the NCBI
+Taxonomy browser is [here](https://www.ncbi.nlm.nih.gov/taxonomy). The IDs for
+currently permitted species are **NCBI:txid10090** for mouse, 
+**NCBI:txid9606** for human, and **NCBI:txid9483** for marmoset.
+
+#### **Parent Specimen ID** (string, optional)
+* The unique identifier of the parent specimen associated with this
+sample, if applicable.
+
+#### **Parent Specimen Type** (string, optional)
+* The specimen type of the parent specimen associated with this
+sample, if applicable. The list of permitted values are found in the 
+`Specimen Type` box in the `Controlled Values` tab.
+
+#### **Donor ID** (string, optional)
+* The unique identifier of the individual donor providing the biological
+sample.
+
+#### **Age** (string)
+* The age of the subject when the sample was taken. Please also add 
+the relevant units to the value (e.g., '32d')
+
+#### **Sex** (string)
+* The sex of the subject. Values are 'male' or 'female'.
+
+#### **Genotype** (string, optional)
+* The genotype of the animal. Entries should be formatted as
+"allele1/allele2;allele3/allele4".
+
+#### **Modality** (string)
+* The type of data modality. These are broad terms used to categorize the data
+at a high level. The list of permitted values can be found on the `Controlled
+Values` sheet.
+
+#### **Technique** (string)
+* The experimental technique that generated this sample. This differentiates 
+between data types within a modality. The list of permitted values are found in the 
+`Controlled Values` tab.
+
+#### **Anatomical Structure** (string, optional)
+* The name of anatomical structure from which sample was obtained. The
+`Controlled Values` sheet has links to brain atlas references for mouse
+and human structures - fill in this entry with the (acronym/name) field
+for the relevant brain area. For marmoset, use the closest analogous
+human structure.
+
+#### **Subspecimen Type** (string)
+* The type of subspecimen counted. See the `Controlled Values` tab
+for allowed values.
+
+#### **Total Processed Subspecimens** (integer)
+* Total number of quality-controlled subspecimens from this sample (e.g.,
+the number of cells from this sample; if there are no subspecimens,
+this may equal 1).
```

Comment:
As above, I think that it would make more sense if this was an optional field and it was left blank if there was no subspecimens. If we want it to be required, then putting 0 subspecimens makes more sense than 1 subspecimen. How would we differentiate from a sample with 1 true processed subspecimen otherwise?